packages/cli/README.md

```
@@ -11,32 +11,71 @@
+  <a href="https://www.npmjs.com/package/@faststore/cli">
+    <img src="https://badge.fury.io/js/%40faststore%2Fcli.svg" alt="npm version" />
+  </a>
+</p>
+
+`@faststore/cli` is the primary toolchain for FastStore projects. It handles project initialization, local development, builds, GraphQL schema caching, and CMS synchronization. It also keeps store projects in sync with `@faststore/core`.
+
+
+## Package structure
 
-The FastStore Command Line Interface (CLI) is the main pre-configured tool for initializing, building and developing FastStore projects.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)